docs/build/guides/dapps/soroban-contract-init-template.mdx

```diff
@@ -1,442 +1,282 @@
 ---
-title: Develop contract initialization templates
-hide_table_of_contents: true
-description: Create frontend templates that can be used to work with Stellar smart contracts in frontend applications
+title: Develop contract initialization frontend templates
+description: Understand, find, and create your own frontend templates for use with Stellar CLI's `stellar contract init` command
 ---
 
 <head>
-  <title>Develop Soroban contract Initialization Template</title>
+  <title>Develop contract initialization frontend templates</title>
   <meta charSet="utf-8" />
   <meta
     property="og:title"
-    content="Develop contract Initialization templates"
+    content="Develop contract initialization frontend templates"
   />
   <meta
     property="og:description"
-    content="This guide will walk you through the process of developing soroban contract templates for working with soroban smart contracts in frontend applications. This guide will cover the process of creating or developing the templates itself as well as how to work with the template to build, deploy, and generate smart contract bindings for the template"
+    content="Understand, find, and create your own frontend templates for use with Stellar CLI's `stellar contract init` command"
   />
 </head>
 
-This guide will walk you through the process of developing contract templates for working with Stellar smart contracts in frontend applications. This guide will cover the process of creating or developing the templates itself as well as how to work with the template to build, deploy, and generate smart contract bindings for the template.
+This guide picks up where [Build a Dapp Frontend](../../apps/dapp-frontend.mdx) left off, taking a deeper look at the `--frontend-template` used there. From there, we'll:
 
-## Prerequisites
+1. Search GitHub for other Soroban templates
+1. Build our own simple template
 
-Before you begin, ensure you have the following installed:
+Building our own template will be a great way to learn how they work. They're not that complicated!
 
-1. [Rust](https://www.rust-lang.org/) and Cargo (for compiling smart contracts)
-2. [Node.js](https://nodejs.org/en) and npm (for running JavaScript deployment scripts)
-3. [Stellar CLI](../../smart-contracts/getting-started/setup.mdx#install-the-stellar-cli)
-4. [React](https://react.dev/learn)
+## Search GitHub for other Soroban templates
 
-## Anatomy of contract initialization (init) template
+The official template maintained by Stellar Development Foundation, as used in [Build a Dapp Frontend](../../apps/dapp-frontend.mdx), lives on GitHub at [stellar/soroban-template-astro](https://github.com/stellar/soroban-astro-template). It uses the [Astro](https://astro.build/) web framework. While Astro works with React, Vue, Svelte, and any other UI library, the template opts not to use them, preferring Astro's own templating language, which uses vanilla JavaScript with no UI library.
```

Comment:
Should we link to this guide at the end of the Dapp Frontend tutorial?

Reply:
I also

- in "Build a Dapp: Wrapping up": removed the link to `soroban-example-dapp`, which is quite outdated at this point
- in this new file: added a "Wrapping up" section to the end of this new guide, which links people to the same things as "Build a Dapp: Wrapping up"

lmk if this all looks good!